google/cloud/bigquery/table.py

```diff
@@ -1789,6 +1789,14 @@ class RowIterator(HTTPIterator):
             The query text used.
         total_bytes_processed (Optional[int]):
             total bytes processed from job statistics, if present.
+        slot_millis (Optional[int]):
+            Number of slot ms the user is actually billed for.
+        created (Optional[datetime.datetime]):
+            Datetime at which the job was created.
+        started (Optional[datetime.datetime]):
+            Datetime at which the job was started.
+        ended (Optional[datetime.datetime]):
+            Datetime at which the job finished.
```

Comment:
RowIterator is also used for `list_rows(table)`, so let's clarify that this is for the associated query, if present.

```suggestion
        total_bytes_processed (Optional[int]):
            If representing query results, the total bytes processed by the associated query.
        slot_millis (Optional[int]):
            If representing query results, the number of slot ms billed for the associated query.
        created (Optional[datetime.datetime]):
            If representing query results, the creation time of the associated query.
        started (Optional[datetime.datetime]):
            If representing query results, the start time of the associated query.
        ended (Optional[datetime.datetime]):
            If representing query results, the end time of the associated query.
```